modules/CONT-02_Using-branches.yml

```diff
@@ -26,10 +26,10 @@ screens:
         - do: Go to another repo and then return to the class repo
           say: "However, a word of caution. When you leave the repository and come back, notice that GitHub automatically assumes you want to see the items on the master branch."
         - do: Click the branch dropdown and reselect the branch
-          say: "If you want to continue working on your branch, you will need to reselect it using the branch dropdown."
+          say: "If you want to continue working on your branch, you will need to reselect it using the branch dropdown. On the next page you will create your own branch."
```

Comment:
@loranallensmith I typically use "page" when it is something they will click "next" to reach and "section" when it is something they will scroll down to see. I considered saying "Next you will create your own branch" but then I was afraid they would try to create their branch without clicking next to see the instructions for the activity. I am open to suggestions and ideas here - it is quite the conundrum.